ivadomed/preprocessing.py

```diff
@@ -0,0 +1,46 @@
+import copy
```

Comment:
I am wondering if we should not move these to `transforms.py` as there are only functions that split / apply transforms. Not too sure. Happy to discuss :-)